backend/app/controllers/spree/admin/option_types_controller.rb

```diff
@@ -21,7 +21,7 @@ def location_after_save
       end
 
       def load_product
-        @product = Spree::Product.find_by_param!(params[:product_id])
+        @product = Spree::Product.find_by!(param: params[:product_id])
```

Comment:
This method is defined by us in `core/lib/spree/core/permalinks.rb` and are included in all active record classes. This should not be replaced.